### ISSUE TITLE
Проблемы с авторизацией с компьютеров с одним белым IP-адресом

### ISSUE THREAD

**Issue by hgenru** [pythontutor/pythontutor-ru#80]
Итак. Есть класс полный компьютеров. Все они сидят в одной локальной сети и в интернет смотрят через общий ip-адрес.

Проблема в том, что сайт каким-то образом привязывает авторизацию к IP и когда кто-то пытается зайти на сайт с компьютера, с которого еще никто туда не заходил сайт думает, что произведен вход в последний активный аккаунт.

Соответственно выйти из него тоже не так просто.

**Comment by vpavlenko** [comment]
Насколько я понимаю, регистрация всё же не привязана к IP, а привязана к
сессии, которая хранится на конкретном компьютере в куках браузера. Вы
уверены, что с того компьютера, на котором возникает нежелательный вход в
последний аккаунт, до этого никто не пользовался сайтом?

В качестве обхода проблемы предлагаю попробовать:
- использовать режим инкогнито браузера
- использовать прокси-плагины типа ZenMate
6 февр. 2015 г. 7:22 пользователь "Alexander Plesovskikh" <
notifications@github.com> написал:

**Comment by hgenru** [comment]
@vpavlenko да, я сам удивился. Я чистил куки и даже пробовал заходить с "чистого" компьютера из той же локальной сети.

Симптомы очень странные.
